applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp

```diff
@@ -252,7 +252,7 @@ void EmbeddedSkinVisualizationProcess::CopyOriginNodalValues(){
     ModelPart::NodeConstantIterator old_nodes_begin = mrModelPart.NodesBegin();
     ModelPart::NodeIterator new_nodes_begin = mrVisualizationModelPart.NodesBegin();
 
-    #pragma omp parallel for
+    //#pragma omp parallel for
```

Comment:
are u sure than new_nodes and old_nodes have the same size??

Reply:
No. Do you think that this might be the problem?